galpy/potential_src/SpiralArmsPotential.py

```diff
@@ -0,0 +1,584 @@
+                        can be a Quantity with units of density. (amp = 4 * pi * G *rho0)
+            :param normalize: if True, normalize such that vc(1.,0.)=1., or, if given as a number
+                              such that the force is this fraction of the force necessary to make vc(1.,0.)=1.
+            :param ro: distance scales for translation into internal units (default from configuration file)
+            :param vo: velocity scales for translation into internal units (default from configuration file)
+            :param N: number of spiral arms
+            :param alpha: pitch angle of the logarithmic spiral arms in radians (can be Quantity)
+            :param r_ref: fiducial radius where rho = rho0 (r_0 in the paper by Cox and Gomez) (can be Quantity)
+            :param phi_ref: reference angle (phi_p(r_0) in the paper by Cox and Gomez) (can be Quantity)
+            :param Rs: radial scale length of the drop-off in density amplitude of the arms (can be Quantity)
+            :param H: scale height of the stellar arm perturbation (can be Quantity)
+            :param Cs: list of constants multiplying the cos(n*gamma) term in the mass density expression
+            :param omega: rotational speed of the spiral arms (can be Quantity)
+        OUTPUT:
+            (none)
+        HISTORY:
+            2017-05-12  Jack Hong (UBC)
+        """
+        Potential.__init__(self, amp=amp, ro=ro, vo=vo, amp_units=amp_units)
+        if _APY_LOADED:
+            if isinstance(alpha, units.Quantity):
+                alpha = alpha.to(units.rad).value
+            if isinstance(r_ref, units.Quantity):
+                r_ref = r_ref.to(units.kpc).value / self._ro
+            if isinstance(phi_ref, units.Quantity):
+                phi_ref = phi_ref.to(units.rad).value
+            if isinstance(Rs, units.Quantity):
+                Rs = Rs.to(units.kpc).value / self._ro
+            if isinstance(H, units.Quantity):
+                H = H.to(units.kpc).value / self._ro
+            if isinstance(omega, units.Quantity):
+                omega = omega.to(units.km / units.s / units.kpc).value \
+                        / bovy_conversion.freq_in_kmskpc(self._vo, self._ro)
+
+        self._N = -N  # trick to flip to left handed coordinate system; flips sign for phi and phi_ref, but also alpha.
+        self._alpha = -alpha  # we don't want sign for alpha to change, so flip alpha. (see eqn. 3 in the paper)
+        self._sin_alpha = np.sin(-alpha)
+        self._tan_alpha = np.tan(-alpha)
+        self._r_ref = r_ref
+        self._phi_ref = phi_ref
+        self._Rs = Rs
+        self._H = H
+        self._Cs = np.array(Cs)
+        self._ns = np.arange(1, len(Cs) + 1)
+        self._omega = omega
+        self._rho0 = 1 / (4 * np.pi)
+        self._HNn = self._H * self._N * self._ns
+
+        if normalize or (isinstance(normalize, (int, float)) and not isinstance(normalize, bool)):
```

Comment:
I think we should get rid of the ``normalize`` keyword and these lines. Normalizing this potential doesn't make sense.